src/appleseed.studio/python/pythoninterpreter.cpp

```diff
@@ -193,11 +216,8 @@ void PythonInterpreter::load_plugins()
             compute_bundled_plugins_path()));
 }
 
-bpy::object PythonInterpreter::execute(const string& command)
+bpy::object PythonInterpreter::execute(const string& command, const bool notify)
 {
-    if (!m_is_initialized)
```

Comment:
Why is this check no longer required?

Reply:
I think it was redundant before. We always initialize the interpreter.